src/cts/src/TritonCTS.tcl

```diff
@@ -198,7 +198,7 @@ proc clock_tree_synthesis { args } {
 
   cts::set_obstruction_aware [info exists flags(-obstruction_aware)]
 
-  if { [ord::get_db_block] == "NULL" } {
+  if { [ord::get_db_block] == "nullptr" } {
```

Comment:
`NULL` is correct for tcl. `nullptr` is only for C++

Reply:
Fixed